src/main/java/Entities/UserPublicProfile.java

```diff
@@ -1,10 +1,32 @@
 package Entities;
-
 import java.util.HashMap;
-
 public class UserPublicProfile {
-//    private json;
+    private String biography;
+    private HashMap<String, String> preferences;
+
     public UserPublicProfile(){
-//        this.preferences = HashMap<String, String> ;
+        HashMap<String, String> preferences = new HashMap<>();
+        /*Prefilling in the preferences*/
+        preferences.put("courseCode", "");
+        preferences.put("timeCommitment", "");
+        preferences.put("location", "");
+        preferences.put("meetingTime", "");
+        this.preferences = preferences;
+    }
```

Comment:
As Helena commented on Julia's pull request, I believe you guys should coordinate to match the name's of the preferences as it is important for Helena's use case matching algorithm